packages/protocol/contracts/ACE/noteRegistry/interfaces/Behaviour.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+import "../../../interfaces/IAZTEC.sol";
+import "../proxies/BaseAdminUpgradeabilityProxy.sol";
+import "../Manager.sol";
+
+/**
+ * @title NoteRegistryBehaviour contract which contains the storage variables that define the set of valid
+ * AZTEC notes for a particular address
+ * @author AZTEC
+ * @dev TODO
+ * Copyright Spilbury Holdings Ltd 2019. All rights reserved.
+ **/
+contract NoteRegistryBehaviour is Ownable, IAZTEC {
```

Comment:
Is it worthwhile renaming this file from `Behaviour.sol` to `NoteRegistryBehaviour.sol`, to match the Solidity name of this contract/interface? That way when it's imported elsewhere, inherited and referred to as `NoteRegistryBehaviour` rather than `Behaviour` it's a bit clearer as to what contract we're referring to